src/wireviz/wv_helper.py

```diff
@@ -113,6 +114,13 @@ def tuplelist2tsv(inp, header=None):
 def index_if_list(value, index):
     return value[index] if isinstance(value, list) else value
 
+def remove_links(inp):
+    if (inp is None):
+        return None
+    ret = inp.replace("</a>", "")
+    ret = re.sub(r"<a[^>]*>", "", ret)
+    return ret
+
 def html_line_breaks(inp):
     return inp.replace('\n', '<br />') if isinstance(inp, str) else inp
```

Comment:
I suggest this code instead:
```py
return remove_links(inp).replace('\n', '<br />') if isinstance(inp, str) else inp
```
Then most calls to `remove_links()` in `Harness.py` can be removed.